release-engineering/role-handbooks/branch-manager.md

```diff
@@ -373,6 +380,39 @@ The following are some ways to determine if the release process was successful:
 
 3. CHANGELOG-X.Y.md is automatically loaded into the kubernetes/kubernetes repo: [https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.16.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.16.md)
 
+### Image Promotion
+
+ We need publishing images to the `k8s-staging-kubernetes` and then promoting them to prodution. To do that we need to follow the steps:
+
+ This step should be done after `Mock Stage` and `Mock Release` are completed and no errors.
+
+ - Forking/cloning https://github.com/kubernetes-sigs/k8s-container-image-promoter
+ - Forking/cloning https://github.com/kubernetes/k8s.io. If you already have that make sure you have an up-to-date copy of the repository.
+ - From the root of `k8s-container-image-promoter` repository
+
+ ```shell
+ go install ./cmd/cip-mm/... # Installs the manifest merge utility, cip-mm
```

Comment:
Can we `get get …` the promoter? If so, then I would prefer this way.

Reply:
tried but got
```
GO111MODULE=on go get sigs.k8s.io/k8s-container-image-promoter/cmd/cip-mm
go get sigs.k8s.io/k8s-container-image-promoter/cmd/cip-mm: module sigs.k8s.io/k8s-container-image-promoter@upgrade found (v1.0.7), but does not contain package sigs.k8s.io/k8s-container-image-promoter/cmd/cip-mm
```